### PR TITLE
Add LinuxArm64 target to Compose runtime

### PR DESCRIPTION
**Pull request by JakeWharton**
Test: ./gradlew :compose:runtime:runtime:compileKotlinLinuxArm64 :compose:runtime:runtime-saveable:compileKotlinLinuxArm64

### CODE REVIEW

File: compose/runtime/runtime/build.gradle [hunk 1]
```diff
@@ -175,6 +176,8 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
             desktopMain.dependsOn(jbMain)
             jsNativeMain.dependsOn(jbMain)
 
+            linuxArm64Main.dependsOn(nativeMain)
+            linuxArn64Test.dependsOn(nativeTest)
```
Comment (igordmn) [inline compose/runtime/runtime/build.gradle]:
```suggestion
            linuxArm64Test.dependsOn(nativeTest)
```

Comment (JakeWharton) [reply compose/runtime/runtime/build.gradle]: Oof thanks!

Comment (JakeWharton) [reply compose/runtime/runtime/build.gradle]: Had to remove you as a co-author since it used your opaque GitHub email and there's no Google CLA for that.

**Review by eymar — APPROVED** [review]
Thanks!

**Comment by eymar** [comment]
There is an issue with new test report generator. The Github action failure can be ignored for now.

**Pull request merged**